python/pyarrow/tests/test_array.py

```diff
@@ -300,6 +301,8 @@ def test_nulls(ty):
 def test_array_from_scalar():
     today = datetime.date.today()
     now = datetime.datetime.now()
+    now_utc = now.replace(tzinfo=pytz.utc)
```

Comment:
Based on my experimentations, you should write:
```python
now_utc = datetime.datetime.now(tz=pytz.utc)
```

(simply calling `.replace(tzinfo=pytz.utc)` doesn't adjust the recorded time for the timezone change, so you get the local time tagged with a UTC timezone)

And, yes, this probably doesn't matter for the test's correctness :-)